src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -21,6 +24,22 @@ const RenderMatchUp = props => {
   return (
     <>
       <Header displayMenu={true} title="The Matchup" />
+      <QuestionCircleOutlined
+        className="question-icon"
+        onClick={() => {
+          setModalVisible(true);
+        }}
+      />
+      <InstructionsModal
```

Comment:
Great job on utilizing the InstructionsModal component. That is a great way to leverage reusable components.